examples/git.tfnot

```diff
@@ -10,26 +10,28 @@ module "terraform_pipeline" {
   cb_env_type                        = "LINUX_CONTAINER"
   cb_tf_version                      = var.terraform_version
   cb_env_name                        = var.env_name
+  
+  # This is part of an or statement of code star or s3, this section is meant for if your artifacts are GIT and not local. Use whitespace to emulate nulls, they must still be defined.
```

Comment:
\# These inputs should be defined if your source artifacts are in a GitHub or BitBucket Git repository. Use an empty string to emulate any null values; they must still be defined.

Reply:
This indicates to me that there is an opportunity to improve logic in this module.